codespell_lib/data/dictionary.txt

```diff
@@ -25999,7 +26005,11 @@ syatem->system
 syatems->systems
 sybsystem->subsystem
 sybsystems->subsystems
+sychronise->synchronise
```

Comment:
Can you add the -ed and -er variants of the s spelling too please.

Reply:
done